NameLayerMain/src/vg/civcraft/mc/namelayer/database/GroupManagerDao.java

```diff
@@ -291,10 +292,66 @@ public void run() {
 			catch (SQLException e) {
 				
 			}
+			//old permission table is no longer used, so delete it
```

Comment:
Hmm yeah, I agree. I originally put it in the older upgrade, because logic wise it belongs there (that upgrade made that table obsolete), but it really doesnt matter if its deleted later on (or at all, it's just cleanup)